web_notify/models/res_users.py

```diff
@@ -0,0 +1,130 @@
+# pylint: disable=missing-docstring
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+
+DEFAULT_MESSAGE = "Default message"
+
+SUCCESS = "success"
+DANGER = "danger"
+WARNING = "warning"
+INFO = "info"
+DEFAULT = "info"
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    def notify_success(
+        self, message="Default message", title=None, sticky=False, beep=True
+    ):
```

Comment:
I think that beep option should be défaut to False. 
- migrating module that use the crurent module will générate a lot of beep.
- in a multi tab usage it will generate a lot of beep.

Reply:
correct, thanks for your feedback, Fixed.